playlist/models.py

```diff
@@ -20,7 +20,7 @@ class Song(models.Model):
     """
     playlist = models.ForeignKey(Playlist)
     title = models.CharField(max_length=255)
-    link = models.URLField()
+    link = models.CharField(max_length=50)
```

Comment:
Learn to research. we are not making sample projects. have pride in your work

https://stackoverflow.com/questions/6180138/whats-the-maximum-length-of-a-youtube-video-id